packages/core/src/navigation/navigation.element.spec.ts

```diff
@@ -120,11 +120,21 @@ describe('cds-navigation', () => {
       element.dispatchEvent(ee);
     }
 
+    // function enterEvent(element: any) {
```

Comment:
This is part of the TODO code below. It isn't running the lifecycle functions in the same order as it is in the demo (normal browser). I am leaving it for now so I can remember where I was when I come back to these tests.